### DISCUSSION TITLE
Changing Z depth

### DISCUSSION THREAD

**Discussion by dkag123** [winder/Universal-G-Code-Sender#2235] [Help]
Is there a way to change the Z depth somewhat 'on the fly'? I am thinking something similar to how the speed/spindle/rapids can be changed. 
What I am doing now is stopping a job, move the spindle off the stock, go to Z depth, reset it, then start the job again. I am just curious if there is an easier way to do this without all those steps.

**Comment by breiler** [comment]
There is no override for it like when fine tuning the feed and speed under runtime.

You will have to stop the job, change the work coordinate and rerun the job like you described.

But instead of jogging, you can manually change change the work coordinate by clicking the coordinate box. So if it runs too shallow you can set the work coordinate to +1mm which will make it cut one millimeter deeper. 

There is also support for simple mathematic formulas, just use the `#`-character as a placeholder for the current value.

![2023-06-20 16-21-23 2023-06-20 16_22_19](https://github.com/winder/Universal-G-Code-Sender/assets/8962024/4510c1fa-ccd4-4c78-9bf1-0b667c8532c8)

**Reply by dkag123** [reply]
Ah, thanks, I will try it
